lib/api/traversing.js

```diff
@@ -353,6 +353,14 @@ exports.eq = function(i) {
   return this[i] ? this._make(this[i]) : this._make([]);
 };
 
+exports.gt = function(i) {
+  return this.length > 1 ? this._make(this.slice(i + 1)) : this;
```

Comment:
I'm not sure about the behavior for `length <= 1`. Shouldn't an empty collection be returned?